src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -187,6 +192,22 @@ private void updateHearing(HearingWrapper wrapper) throws UpdateCaseException, L
             throw new ListingException("Listing duration must be multiple of 5.0 minutes");
         }
 
+        HearingsGetResponse hearingsGetResponse = hmcHearingApiService.getHearingsRequest(caseId, null);
+        CaseHearing hearing = HearingsServiceHelper.findExistingRequestedHearings(hearingsGetResponse, true);
+        if (nonNull(hearing)) {
+            Long hmcHearingVersionId = getHearingVersionNumber(hearing);
+            HearingDetails caseDataHearing = caseData.getLatestHearing().getValue();
```

Comment:
should we be getting the hearing that matches hearing.getHearingId() from the case rather than getting the latest?